rootfs/etc/profile.d/_40-preferences.sh

```
@@ -25,54 +14,33 @@ fi
-			boot install
-		fi
-		export GEODESIC_CUSTOMIZATION_DISABLED="/localhost not a volume"
-	elif mkdir -p $GEODESIC_CONFIG_HOME; then
-		echo $(yellow Created directory "$GEODESIC_CONFIG_HOME" '(GEODESIC_CONFIG_HOME)')
-	else
-		echo $(red Cannot create directory "$GEODESIC_CONFIG_HOME" '(GEODESIC_CONFIG_HOME)')
-	fi
+[[ -n ${WORKSPACE_MOUNT} ]] || export WORKSPACE_MOUNT=/workspace
+if [[ -z $(find "${WORKSPACE_MOUNT}" -mindepth 1 -maxdepth 1) ]]; then
+	red "########################################################################################" >&2
+	red "# No filesystem is mounted at $(bold ${WORKSPACE_MOUNT}) which limits Geodesic functionality." >&2
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)